pkg/auth/db_service.go

```diff
@@ -4,22 +4,47 @@ package auth
 
 import (
 	"context"
+	"encoding/json"
 	"errors"
 	"fmt"
+	"io"
 	"reflect"
+	"strconv"
 	"time"
 
 	sq "github.com/Masterminds/squirrel"
 	"github.com/georgysavva/scany/pgxscan"
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v4/pgxpool"
 	"github.com/treeverse/lakefs/pkg/auth/crypt"
 	"github.com/treeverse/lakefs/pkg/auth/keys"
 	"github.com/treeverse/lakefs/pkg/auth/model"
 	"github.com/treeverse/lakefs/pkg/auth/params"
 	"github.com/treeverse/lakefs/pkg/db"
+	"github.com/treeverse/lakefs/pkg/kv"
+	kvpg "github.com/treeverse/lakefs/pkg/kv/postgres"
 	"github.com/treeverse/lakefs/pkg/logging"
+	"github.com/treeverse/lakefs/pkg/version"
 	"golang.org/x/crypto/bcrypt"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+//nolint:gochecknoinits
+func init() {
+	tablesToDrop := []string{
```

Comment:
auth_installation_metadata?

Reply:
Currently `MetadataManager` over `KV` is not implemented yet, so migrating it will be meaningless and cannot be tested and verified.
I think we should exclude this from KV2 and open a different issue for that - WDYT?
@itaiad200 - What's your take on that?

Reply:
Moreover - though we can already migrate the table to KV (and have it unused at the moment) dropping it at the end of a successful migration will turn the system unusable

Reply:
I just noticed it's not related to the auth service so disregarded all comments about it. We do need to add it to our plan (I think we missed it)